sw/device/silicon_creator/lib/sigverify_mod_exp.h

```diff
@@ -26,11 +26,27 @@ extern "C" {
  * @param key An RSA public key.
  * @param sig Buffer that holds the signature, little-endian.
  * @param result Buffer to write the result to, little-endian.
- * @return True if successful, false otherwise.
+ * @return The result of the operation.
  */
 rom_error_t sigverify_mod_exp_ibex(const sigverify_rsa_key_t *key,
                                    const sigverify_rsa_buffer_t *sig,
                                    sigverify_rsa_buffer_t *result);
+/**
```

Comment:
Vertical whitespace between function declaration and start of comment block.

Reply:
Done.